.eslintignore

```diff
@@ -1,3 +1,3 @@
 **/node_modules
 **/client/dist/bundle.js
-
+**/spec/unit tests/exampleData.js
```

Comment:
**/spec/unit:
so all things under spec/unit folder will be ignored?

Reply:
it should just be the exampleData.js file that I added (but haven't pushed yet) since I copied postman data and that has double quotes everywhere